dashboard/src/components/DetailedView/DetailedView.js

```diff
@@ -87,39 +89,48 @@ const DetailedView = ({ allPollutants = [] }) => {
       <div className="detailed-container">
         <div className="list-and-detail-container">
           <div className="pollutant-select-container">
-            <PollutantList
-              pollutants={pollutants}
-              onPollutantSelect={id => {
-                const currPollutant = pollutants.filter(pollutant => Number(pollutant.id) === Number(id));
-                setSelectedPollutant(currPollutant[0]);
-              }}
-            />
-
-            <div className="pollutant-details-container">
-              <PollutantDetails pollutant={selectedPollutant} />
-            </div>
+            <PollutantList onPollutantSelect={setSelectedPollutant} />
```

Comment:
Where are you getting the pollutants?

Reply:
There is a pollutant list on the PollutantList.js file!

Reply:
Awesome!